src/overmapbuffer.cpp

```diff
@@ -464,29 +515,35 @@ std::vector<npc*> overmapbuffer::get_npcs_near_player(int radius)
     return get_npcs_near(plpos.x, plpos.y, plpos.z, radius);
 }
 
-std::vector<overmap *> overmapbuffer::get_overmaps_near( point location, int radius )
+std::vector<overmap*> overmapbuffer::get_overmaps_near( point const location, int const radius )
 {
     // Grab the corners of a square around the target location at distance radius.
     // Convert to overmap coordinates and iterate from the minimum to the maximum.
-    std::set<point> distinct_corners;
```

Comment:
What's the rationale for this change? The sort and erase at the end seems quite indirect compared to simply enumerating the overmap coordinates.